### PR TITLE
Run TSLint during builds

### PR DESCRIPTION
**Pull request by petschekr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 sudo: required
 dist: trusty
 
+notifications:
+  email: false
```
Comment (illegalprime) [inline .travis.yml]: Why turn off email notifications? Have you been getting emails?

Comment (petschekr) [inline .travis.yml]: A ton, basically every single time a new branch is built, whenever the build breaks, and whenever the build gets fixed.

Comment (illegalprime) [inline .travis.yml]: we just can't edit `.travis.yml` so we can be sure that we have repeatable builds. I can add a check for changing these sensitive files in the build. But then we should decide if we want emails on or off or optional.

Comment (petschekr) [inline .travis.yml]: The more important thing here is running TSLint during the build so I can take this commit out

Comment (petschekr) [inline .travis.yml]: I would leave it on for now. I really wish Travis supported turning it off from the settings on the website. I'll remove this for now.

Comment (illegalprime) [inline .travis.yml]: mmm yeah, open an issue on hackgt/hackgproject , maybe we'll make a hackgt bot account that gets all these things?

**Comment by petschekr** [comment]
@illegalprime Is this good to merge now without the changes to Travis?

**Comment by illegalprime** [comment]
All good!

**Pull request merged**